src/Analyzers/MSTest.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -1,2 +1,7 @@
 ﻿; Unshipped analyzer release
 ; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+MSTEST0010 | Usage | Info | WorkItemAttributeOnTestMethodAnalyzer
```

Comment:
Let's make this one 8 as it will be the first I merge (after #2176). Also, it seems that the rule is also not handling the `notes` part correctly so we will need to fix it by hand copying the rule description.